src/oop/object_interactions/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,51 @@
+# oop-object_interactions-medium-q2
+
+- Java question
+
+## Question Text
+
+Many people keep secrets that they only share with a trusted friend. Make a class called **Person** that has a secret
+and potentially knows about the secrets of his/her friends. This class has several attributes: **myName**, **mySecret**,
+a list of names of **friends**, and a list of **secrets** from those friends.
+
+## Person Class
+
+The `Person` class currently has the following items:
+
+- Attributes:
+    - A `String` called **myName**
+    - A `String` called **mySecret**
+    - An array of `String` called **friends**
+    - An array of `String` called **secrets**
```

Comment:
these two variable names are inconsistent in this file. Here they are "friends" and "secrets" but farther down it is "friendNames" and "friendSecrets"